modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -8,35 +8,114 @@
 = Creating a KubeletConfig CRD to edit kubelet parameters
 
 The kubelet configuration is currently serialized as an Ignition configuration, so it can be directly edited. However, there is also a new
-`kubelet-config-controller` added to the Machine Config Controller (MCC). This allows you to create a `KubeletConfig` custom resource (CR) to edit the kubelet parameters.
+`kubelet-config-controller` added to the Machine Config Controller (MCC). This allows you to use a `KubeletConfig` custom resource (CR) to edit the kubelet parameters.
+
+You should have one `KubeletConfig` CR for each machine config pool with all the config changes you want for that pool. If you are applying the same content to all the pools, you only need one `KubeletConfig` CR for all the pools. 
+
+You should edit an existing `KubeletConfig` CR to modify existing settings or add new settings instead of creating a new CR for each change. It is recommended to create a new CR only to modify a different machine config pool, or for changes that are intended to be temporary so that you can revert the changes.
+
+You can create multiple `KubeletConfig` CRs, as needed, with a limit of ten per cluster. For the first `KubeletConfig` CR, the MCO creates a machine config appended with `kubelet`. With each subsequent CR, the controller creates a new `kubelet` machine config with a numeric suffix. For example, if you have a `kubelet` machine config with a `-2` suffix, the next `kubelet` machine config is appended with `-3`.
+
+If you want to delete the machine configs, you should delete them in reverse order to avoid exceeding the limit. For example, you should delete the `kubelet-3` machine config before deleting the `kubelet-2` machine config.
+
+[NOTE]
+====
+If you have a machine config with a `kubelet-9` suffix, and you create another `KubeletConfig` CR, a new machine config is not created, even if there are fewer than 10 `kubelet` machine configs.
+====
+
+.Example showing a `KubeletConfig` CR
+[source,terminal]
+----
+$ oc get kubeletconfig
+----
+
+[source, terminal]
+----
+NAME                AGE
+set-max-pods        15m
+----
+
+.Example showing a `KubeletConfig` machine config
+[source,terminal]
+----
+$ oc get mc | grep kubelet
+----
+
+[source, terminal]
+----
+...
+99-worker-generated-kubelet-1                  b5c5119de007945b6fe6fb215db3b8e2ceb12511   3.2.0             26m
+...
+----
+
+.Prerequisites
+
+. Obtain the label associated with the static `MachineConfigPool` CR for the type of node you want to configure.
```

Comment:
This is sort of a single-step procedure, though it has two options as substeps. Maybe make them all unordered `*` and `**` instead of ordered `.` and `..`?

Reply:
I copied this from another module in the assembly. Maybe I can change both.